scripts/install_linux_dependencies.sh

```diff
@@ -3,23 +3,23 @@
 echo "Preparing dependencies for linux/ubuntu"
 
 if [ "$(grep -Ei 'debian|buntu|mint' /etc/*release)" ]; then
-   sudo apt update
-   sudo apt install -y libgmp-dev  libssl-dev  make gcc g++
+   apt update
+   apt install -y libgmp-dev  libssl-dev  make gcc g++
 else
-   sudo yum install glibc-static gmp-devel gmp-static openssl-libs openssl-static gcc-c++
+   yum install glibc-static gmp-devel gmp-static openssl-libs openssl-static gcc-c++
 fi
 
 # Prepare temp directory
 mkdir -p temp
 pushd temp
 
 # Prepare harmony dependencies
-git clone --depth=1 https://github.com/harmony-one/bls.git
+git clone https://github.com/harmony-one/bls.git
```

Comment:
why need it?

Reply:
> why need it?

Harmony chain depends on bls signature. We supported this chain.